cypress/support/commands.ts

```diff
@@ -14,23 +14,44 @@ import { Result } from "axe-core";
 import "@cypress/code-coverage/support";
 import "cypress-axe";
 
+const loginWithRetry = (iteration: number = 0): void => {
+    if (iteration >= 4) {
+        // This only ever seems to happen on GH Actions on the first test, so we can just log and move on
+        // Redirects are tested by the auth spec
+        cy.log("Login redirect failed");
```

Comment:
I see this being very easily missed - is there a way to make it a warning?

Reply:
Unfortunarely cy.warn isn't a thing, it's either this or console.warn but the latter isn't guaranteed to be displayed.